router/metrics.go

```diff
@@ -0,0 +1,21 @@
+package router
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"net/http"
+)
+
+// Pump message send counter
+var pumpMsgSend *prometheus.CounterVec
+
+func init() {
+	pumpMsgSend = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "pump_msg_send",
```

Comment:
All prometheus counter metrics should end in `_total`. Personally would call it `logstash_sent_messages_total` or `logstash_messages_sent_total`.